templates/taxi/car_detail.html

```diff
@@ -11,12 +11,23 @@ <h1>
       Update
     </a>
   </h1>
+  <h2>Drivers</h2>
+  <form method="post" action="{% url 'taxi:car-add-or-delete-driver' car.id %}">
```

Comment:
Ensure that the form's action attribute is correctly set to the URL that handles form submissions. Verify that the URL 'taxi:car-add-or-delete-driver' is correctly configured to handle the form submission for adding or deleting a driver from the car.